### DISCUSSION TITLE
How can I increase session length?

### DISCUSSION THREAD

**Discussion by vdrover** [alexjustesen/speedtest-tracker#561] [Q&A] [answered]
Is there a way to customize the session length? Currently I am logging in every day, I'd like to customize how soon before my login session expires.

**Comment by alexjustesen** [comment, marked as answer]
As an environment variable you can pass `SESSION_LIFETIME` to the container or update `SESSION_LIFETIME` in the `.env` file located in your mounted storage location.

**Comment by vdrover** [comment]
In seconds I presume?

**Reply by alexjustesen** [reply]
Correct
